doc/man3/X509_STORE_CTX_get_error.pod

```diff
@@ -103,6 +103,8 @@ the operation was successful.
 
 the issuer certificate of a locally looked up certificate could not be found.
 This normally means the list of trusted certificates is not complete.
+To enable using trusted self-issued but not self-signed certificates
+the B<X509_V_FLAG_PARTIAL_CHAIN> flag may be set.
```

Comment:
This text is wrong.  With partial chain, the first CA in the trust store terminates the chain, it need not be self-issued or self-signed, it just needs to be in the trust store.

Reply:
The point is that if it's self-issued but not self-signed and in the trust store, you get this error if you don't use partial chain.

Reply:
Exactly, the new sentence is meant as a hint to users who wonder why their self-issued trusted cert is not accepted to terminate a cert chain. Or how to phrase this better and avoid misunderstandings?

Reply:
@vdukhovni, I've re-phrased the new sentence for clarity as:
```
To enable using also (not self-signed but) self-issued trusted certificates
the B<X509_V_FLAG_PARTIAL_CHAIN> flag may be set.
```
Ok this way?

Reply:
This is still wrong. With partial_chain any certificate from the trust store (even one that is not self-issued and not self-signed) terminates the chain.  There's no need to drag self-issued into this discussion.